Audacia.Log.AspNetCore/README.md

```diff
@@ -67,7 +67,55 @@ public IConfiguration Configuration { get; set; }
 public void ConfigureServices(IServiceCollection services)
 {
 	services.ConfigureActionContentLogging(Configuration);
-	services.AddControllers(x => x.Filters.Add<LogActionFilterAttribute>());
+    services.AddClaimsTelemetry();
+	services.AddControllers(x => x.Filters.Add<LogClaimsActionFilterAttribute>());
+}
+```
+
+#### AddClaimsTelemetry()
+This extension method is overloaded and if you want to pass in your own implementation of "CustomAdditionalClaimsTelemetryProvider" to select which properties you would like to pass on, do it like below:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+	services.ConfigureActionContentLogging(Configuration);
+    services.AddClaimsTelemetry(new CustomAdditionalClaimsTelemetryProvider((user) => 

```

Comment:
`question:` can the developer do just `services.AddClaimsTelemetry()` without providing a `CustomAdditionalClaimsTelemetryProvider`?

Reply:
If I've understood this correctly, you can (great!), can we document this 'happy path' in which case?

Reply:
Yes you can, I think it's probably not clear in the readme